lib/std/crypto/ecdsa.zig

```diff
@@ -1431,13 +1561,24 @@ test "Test vectors from Project Wycheproof - EcdsaP384Sha384" {
         .{ .key = "04fb01baad5f0b8f79b9cd104d12aab9310146add7d6b4c022d87ae6711178b94d618ca7b3af13854b1c588879e877b336ffffffffdf74c0a52c4c6c8533629f933a131354b58fe08a12bd6a815b287a71cc0a3d92951df5633325a96798ff294b", .msg = "4d657373616765", .sig = "306602310090b95a7d194b73498fba5afc95c1aea9be073162a9edc57c4d12f459f0a1730baf2f87d7d6624aea7b931ec53370fe47023100cbc1ef470e666010604c609384b872db7fa7b8a5a9f20fdefd656be2fcc75db53948102f7ab203ea1860a6a32af246a1", .result = .valid },
         .{ .key = "04fb01baad5f0b8f79b9cd104d12aab9310146add7d6b4c022d87ae6711178b94d618ca7b3af13854b1c588879e877b336ffffffffdf74c0a52c4c6c8533629f933a131354b58fe08a12bd6a815b287a71cc0a3d92951df5633325a96798ff294b", .msg = "4d657373616765", .sig = "3066023100dd4391ce7557cbd005e3d5d727cd264399dcc3c6501e4547505b6d57b40bbf0a7fac794dcc8d4233159dd0aa40d4e0b9023100a77fa1374fd60aa91600912200fc83c6aa447f8171ecea72ae322df32dccd68951dc5caf6c50380e400e45bf5c0e626b", .result = .valid },
     };
+
     for (vectors) |vector| {
         if (tvTry(EcdsaP384Sha384, vector)) {
             try std.testing.expect(vector.result == .valid or vector.result == .acceptable);
         } else |_| {
             try std.testing.expectEqual(vector.result, .invalid);
         }
     }
+
+    if (false) { // non-critical test vectors, skipped because they slow down CI too much
```

Comment:
Would it make sense to add some `extensive_tests` flag to the std lib testing facilities, or to `std.testing` in general?
It might still make sense to run them manually before releases / release candidates. As-is `if (false)` somewhere in `std` are much harder to track/remember than a global flag/mode documenting the intent.